native/components/message-markdown.react.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import { Component } from 'react';
+import * as React from 'react';
+import { View, ViewPropTypes } from 'react-native';
+import { merge, pick } from 'lodash';
+import PropTypes from 'prop-types';
+import SimpleMarkdown from 'simple-markdown';
+
+import styles from '../themes/markdown-styles';
+import darkStyles from '../themes/markdown-dark-styles';
+import rules from './message-markdown-rules.react';
+
+type Props = {
+  onLink: (url: string) => Promise<any>,
+  styles: typeof styles,
+  rules: any,
+  darkStyle: boolean,
+  children: React.Node,
+};
+class Markdown extends Component<Props> {
```

Comment:
Always prefer `React.PureComponent` in this project unless there's an explicit reason otherwise